lib/async/debug/selector.rb

```diff
@@ -67,7 +73,7 @@ def wakeup
 			
 			def close
 				if @monitors.any?
-					raise RuntimeError, "Trying to close selector with active monitors: #{@monitors.values.inspect}! This may cause your socket or file descriptor to leak."
+					raise LeakError
```

Comment:
You need to pass in `@monitors.values`, i.e. `raise LeakError, @monitors.values`.

Reply:
Oops, my bad. 😅